### ISSUE TITLE
Support for update_seq option in allDocs/views

### ISSUE THREAD

**Issue by jareware** [pouchdb/pouchdb#5593]
When requesting a `GET /my-db/_all_docs?update_seq=true`, CouchDB will give you the seq value associated with the result set. This seq value would be useful in a subsequent `GET /my-db/_changes?since=` request, to ensure we get changes _exactly_ since our bulk fetch (`since=now` has a tiny race condition when used for this purpose).

This could probably be done with `ajax` options too, but then wouldn't work against non-HTTP DB's (such as in-browser ones).

What do you think?

**Comment by nolanlawson** [comment]
Sounds like a reasonable feature to me!

**Comment by dharders** [comment]
Will this `update_seq` value be available through replication too ?

i.e. It would be nice to be passed the __remote__ `update_seq` within the `replication.on('change', (info))` callback for calculating replication **progress complete %**.

Currently it's possible but it involves an extra `remoteDB.info()` to get the `update_seq`. It would be great to avoid the additional XHR everytime.

**Comment by willholley** [comment]
@dharders I don't think PouchDB has this info so, without the remote database providing the current update_seq in the changes response, it would have to make the remote call anyway (though maybe only once per batch instead of once per change). CouchDB 2.0 has a `pending` field in the `_changes` response which is the count of changes remaining in the feed - perhaps that would be a useful substitute? If so, can you create a separate ticket?

**Comment by dharders** [comment]
Thanks @willholley I did not know about the `pending` field in 2.0. That should do the job.

New ticket created #5710

**Comment by NNemec** [comment]
This is exactly the feature I had been looking for! Hope to see this in PouchDB!

**Comment by daleharvey** [comment]
Closing as dupe https://github.com/pouchdb/pouchdb/issues/6230
